game-core/src/main/java/org/triplea/game/server/HeadlessGameServer.java

```diff
@@ -36,28 +34,25 @@
 import games.strategy.engine.framework.AutoSaveFileUtils;
 import games.strategy.engine.framework.GameRunner;
 import games.strategy.engine.framework.ServerGame;
-import games.strategy.engine.framework.headless.game.server.ArgValidationResult;
-import games.strategy.engine.framework.headless.game.server.HeadlessGameServerCliParam;
 import games.strategy.engine.framework.startup.mc.GameSelectorModel;
 import games.strategy.engine.framework.startup.mc.ServerModel;
 import games.strategy.engine.framework.startup.ui.ISetupPanel;
+import games.strategy.engine.framework.system.SystemProperties;
 import games.strategy.net.INode;
 import games.strategy.net.IServerMessenger;
 import games.strategy.sound.ClipPlayer;
 import games.strategy.triplea.Constants;
 import games.strategy.triplea.settings.ClientSetting;
 import games.strategy.util.ExitStatus;
 import games.strategy.util.Interruptibles;
-import games.strategy.util.Util;
 import lombok.extern.java.Log;
 
 /**
  * A way of hosting a game, but headless.
  */
 @Log
 public class HeadlessGameServer {
-  private static final int LOBBY_RECONNECTION_REFRESH_SECONDS_MINIMUM = 21600;
-  private static final int LOBBY_RECONNECTION_REFRESH_SECONDS_DEFAULT = 2 * LOBBY_RECONNECTION_REFRESH_SECONDS_MINIMUM;
+  private static final int LOBBY_RECONNECTION_REFRESH_SECONDS_DEFAULT = (int) TimeUnit.HOURS.toSeconds(12);
```

Comment:
For future reference: 12 hours is definitely too long

Reply:
I was thinking the same, let's follow up in: https://github.com/triplea-game/triplea/issues/4552  "lobby.game.reconnection time does not seem right"